pkg/client/client_ops.go

```diff
@@ -17,7 +17,7 @@ func (c *OpClient) ParseOps(
 ) ([]*RosettaTypes.Operation, error) {
 	var ops []*RosettaTypes.Operation
 
-	if tx.Receipt.Type == L1ToL2DepositType && len(tx.Trace) > 0 && tx.Transaction.IsSystemTx() {
+	if tx.Receipt.Type == L1ToL2DepositType && len(tx.Trace) > 0 {
```

Comment:
Can you explain why we remove `tx.Transaction.IsSystemTx()`? If it's system tx we shouldn't go through the fee parser below as it won't cost fee